cmd/drand-cli/cli.go

```diff
@@ -237,12 +237,6 @@ var hashOnly = &cli.BoolFlag{
 	Usage: "Only print the hash of the group file",
 }
 
-var hashInfoReq = &cli.StringFlag{
-	Name:     "chain-hash",
```

Comment:
chain hash is going to be more secure than beaconID for identifying a chain though, right? if i use a short name, the node could give me an arbitrary chain since i won't be able to locally map that to a trust root at this point, right?

Reply:
I don't see any "security" differences, and all the other (even internal) APIs are relying on `beaconID`, so it didn't really make sense to use chain-hash only there, IMO. This is really the only place (to follow another node) where we would need the chain hash so far.

Reply:
this is a place where you won't already have a group file / root of trust already established, right?
in most other places you already have a local mapping of your beaconID back to the group info including the local chain hash, so you aren't needing to fetch (and trust) the chain hash from another node. the tofu of that mapping comes from the manual review and inclusion of the previous group file you load when participating in an initial resharing.

Reply:
I see what you mean. Yeah, to check the integrity of the info it would be best to have it there, indeed. I'll add it back but won't use it at the networking level, only locally to verify integrity.